myuw/static/vue/components/_common/visual_schedule/schedule.vue

```diff
@@ -53,9 +53,14 @@ export default {
     'uw-card': Card,
     'uw-schedule-tab': ScheduleTab,
   },
+  props: {
+    term: {
```

Comment:
Let's change "term" to "quarter" to match with the quarter attribute in term data.
we want add in ...mapState({
 term: (state) => state.visual_schedule.term,